src/applications/vaos/tests/appointment-list/components/ConfirmedAppointmentDetailsPage/index.unit.spec.jsx

```diff
@@ -781,6 +786,7 @@ describe('VAOS <ConfirmedAppointmentDetailsPage>', () => {
   });
 
   it('should verify VA in person calendar ics file format', async () => {
+    const startDateTime = moment();
```

Comment:
Could you use [`date-fns`](https://date-fns.org/) here instead?

Reply:
Is there a plan for getting the platform code migrated to `date-fns`? I evaluated using `date-fns` in VAOS and found it to be somewhat difficult for our use cases. I deprioritized any work switching over to it until we had a solid idea of when the platform would be migrated off of it, since it didn't seem worth the additional complexity and codebase inconsistency for my team until we could actually remove it from our app bundle.

Reply:
We don't have a set plan for migrating the platform code yet, besides asking teams to at least start using it in new code. I won't hold up this PR for this change, if you'd rather stick with moment for now, though. Out of curiosity, do you have any info on where you had difficulties with date-fns? If you want to move this conversation to Slack (and out of this PR, since it's slightly unrelated), that's fine.

Reply:
Here's my writeup: https://github.com/department-of-veterans-affairs/va.gov-team/issues/22635#issuecomment-820536173

It's pretty specific to VAOS, I think, since we deal with timezones in a way that most apps don't have to. Any library that uses the native Date object is going to give VAOS trouble.